### PR TITLE
resolved Form issues/ autocompletion, key stuff

### PR DESCRIPTION
**Pull request by LouieAldana**
resolved address autocomplete, hidden key, rider_radius,etc

### CODE REVIEW

File: starter-code/client/src/pages/Ride.js [hunk 1]
```diff
@@ -155,17 +295,22 @@ export default function Ride() {
     setErrorMsgs(errors);
 
     const objectKeys = Object.keys(errors);
-    const isError = objectKeys.some((error) => {
-      return errors[error] != "";
+    console.log({ objectKeys });
```
Comment (LucienLuc) [inline starter-code/client/src/pages/Ride.js:298]: Remove this line.

**Review by LucienLuc — CHANGES_REQUESTED** [review]
Looks like the ListObject.js file has a lot of changes but its all formatting? Is that correct? Also we might be concerned about the API key exposure from the frontend - see slack message for more information. We can decide what we want to do though.

**Comment by robertgee17** [comment]
i think now that i merged my changes to listobject, the formatting changes from this version cause a merge conflict

File: starter-code/client/src/pages/Ride.js (hunk 1 above)
Comment (LouieAldana) [reply starter-code/client/src/pages/Ride.js:298]: got it

**Review by LucienLuc — APPROVED** [review]

**Pull request merged**